dpe/src/commands/mod.rs

```diff
@@ -148,13 +148,23 @@ pub struct CommandHdr {
 impl CommandHdr {
     const DPE_COMMAND_MAGIC: u32 = u32::from_be_bytes(*b"DPEC");
 
-    pub fn new_for_test(cmd_id: u32) -> CommandHdr {
+    pub fn new_for_test(profile: DpeProfile, cmd_id: u32) -> CommandHdr {
```

Comment:
nit: SInce there's nothing test-specific about this, maybe we should just call the function `new`

Reply:
That's better, done.